client/src/chatCommands.ts

```diff
@@ -109,7 +109,7 @@ const setLeader = (_room: string, args: string[]) => {
 chatCommands.set('setleader', setLeader);
```

Comment:
Since you're in the area, might as well end a hypo if in progress? 
#1260

Reply:
Oh seems like you took care below. Nevermind

Reply:
Intentionally not done. You can pass lead to someone else and have them continue your hypo, it works fine